pkg/command/plugin.go

```diff
@@ -234,7 +235,7 @@ func newDescribePluginCmd() *cobra.Command {
 
 func newInstallPluginCmd() *cobra.Command {
 	var installCmd = &cobra.Command{
-		Use:   "install [name]",
+		Use:   "install " + pluginNameCaps,
```

Comment:
This is not correct.  We can do `plugin install --group` (without a name) or `tanzu plugin install all` (with the all string).

I suggest `Use: "install [PLUGIN_NAME | all]"`